ui.apps/src/main/content/jcr_root/apps/core/cif/components/structure/page/v1/page/_cq_dialog/.content.xml

```diff
@@ -24,27 +24,55 @@
                         jcr:title="Commerce"
                         sling:orderBefore="cloudservices"
                         sling:resourceType="granite/ui/components/coral/foundation/fixedcolumns">
-                        <granite:rendercondition
```

Comment:
On content pages the `associatedContent` section will be shown. Otherwise I think every page that uses the cif page component should be displaying something, right?

Reply:
OK, we need the commerce tab for content pages too. :-)

Reply:
I think so.